Samples/word-add-in-get-set-edit-openxml/README.md

```diff
@@ -0,0 +1,113 @@
+
+After you paste or edit the content in the text area of the task pane, you can click in a blank area of the document and then choose the **Insert…** button on the task pane to test the integrity of your markup. That button uses the JavaScript **setSelectedDataAsync** method to insert the contents of the task pane text area as rich content in Word, using the ooxml coercion type.
+
+## Key components of the sample
+
+The sample contains:
+
+- The WD_OpenXML_js project, which contains:
+    - The WD_OpenXML_js.xml manifest file
+    - The SampleDoc.docx document, which is prepopulated with various types of rich content
+- The WD_OpenXML_js Web project, which contains multiple template files. However, the two files that have been developed as part of this sample solution include:
+    - WD_OpenXML_js.html (in the Pages folder). This contains the HTML user interface that is displayed in the task pane. It consists of two HTML buttons that extract and insert Office Open XML, a DIV where status messages will be written, a textarea HTML control that is used to show you Office Open XML markup, and instructional text
+    - WD_OpenXML_js.js (in the Scripts folder). This script file contains code that runs when the add-in is loaded. The code checks for which APIs are supported by the host application. If the host is 2013, the application uses the common Office.js API. If the host is Word 2016 or later, the add-in uses the Word Javascript API. This startup wires up the **onclick** event handlers for the two buttons in WD_OpenXML_js.html. One of these buttons retrieves the selected document content as Office Open XML, and the other button inserts content into the document via OOXML coercion, using the contents of the text area in the task pane. All other files are automatically provided by the Visual Studio project template for an Office add-in, and they have not been modified in the development of this sample app
+
+**Note**
+
+When you use JavaScript to generate the Office Open XML markup for selected content, it creates an entire document package, which is a far larger payload than you need for inserting just your content. For help interpreting, editing, and simplifying your work with Office Open XML for a Word Add-in, see [Create Better Add-ins for Word with Office Open XML](https://docs.microsoft.com/office/dev/add-ins/word/create-better-add-ins-for-word-with-office-open-xml).
+
+## Troubleshooting
+
+If the Add-in starts with a blank document instead of the one shown in Figure 1, ensure the **StartAction** property of the WD_OpenXML_js project is set to SampleDoc.docx and not just to Word.
```

Comment:
```suggestion
If the add-in starts with a blank document instead of the one shown in Figure 1, ensure the **StartAction** property of the WD_OpenXML_js project is set to SampleDoc.docx and not just to Word.
```
Figure 1a or 1b?